CHANGELOG.md

```diff
@@ -22,6 +39,7 @@
 - Do not encode strings in-place [\#147](https://github.com/ably/ably-ruby/pull/147) ([mattheworiordan](https://github.com/mattheworiordan))
 - Only resume if connection is fresh \(RTN15g\*\) [\#146](https://github.com/ably/ably-ruby/pull/146) ([mattheworiordan](https://github.com/mattheworiordan))
 - Fix channel history pagination test [\#143](https://github.com/ably/ably-ruby/pull/143) ([funkyboy](https://github.com/funkyboy))
+- New release v1.0.5 [\#142](https://github.com/ably/ably-ruby/pull/142) ([funkyboy](https://github.com/funkyboy))
```

Comment:
Why is this new?

Reply:
[It's a PR that's been closed](https://github.com/ably/ably-ruby/pull/142) but the change log generator picked it up as a bug fix. Will remove it.